proposals/0101-halve-rent.md

```diff
@@ -0,0 +1,80 @@
+---
+simd: '0101'
+title: Halve Rent
+authors:
+  - Brooks Prumo (Solana Labs)
+category: Standard
+type: Core
+status: Draft
+created: 2024-01-05
+feature: https://github.com/solana-labs/solana/issues/34656
+---
+
+## Summary
+
+Halve rent.
+
+## Motivation
+
+Projects that build on Solana face operational cost volatility if they
+manage/create accounts for their users.  This is because creating accounts
+incurs *rent*, and that cost is both (1) fixed, and (2) denominated in SOL.
+
+The original cost of rent was based on calculations assuming 1 SOL == 1 USD.
+As the SOL token accrues more value, one side effect is that rent increases
+*as a function of fiat*.  Since not all projects have treasuries exclusively
+in SOL, this can play out where project growth/user acquisition becomes more
+and more expensive over time.
+
+The cost of rent can be reduced to offset user acquisition expenses.
+
+## Alternatives Considered
+
+As noted in the Motivation, rent is both fixed and denominated in SOL.  This
+SIMD changes neither of those two facts, but a longer-term project is in the
+works that addressed the 'fixed' nature of rent.  Ideally the protocol should
+price state based on current usage and demand—incentivizing users to close
+unused accounts, and charging users proportionally, based on demand, for new
+state.  That project will take much longer to design and implement; in the
+meantime, steps can still be made to help projects grow.
+
+## New Terminology
+
+None.
+
+## Detailed Design
+
+A new feature-gate will be added that halves Rent's lamports-per-byte-year.
+
+## Impact
+
+The cost of rent per account will be halved.
+
+Validators will *not* be economically impacted.  Since accounts must carry a
+minimum balance to be rent-exempt, this means there are no rent-paying
+accounts.  Thus, there are no rent fees that validators collect, and changing
+the rent amount will not change that.
+
+Developers *will* be impacted, if they create accounts.  Now, these accounts
+will require half as much rent, in SOL, as before.
+
+## Security Considerations
+
+What security implications/considerations come with implementing this feature?
```

Comment:
Is a potential drawback decreasing "rent" (I'll refer to it as storage cost) that it become potentially dangerous to increase again in the future? I imagine there are checks in the runtime ensuring that an account's lamport balance is always higher than the required threshold or else it gets garbage collected. The nice part of lowering this value is that the threshold requirements for legacy accounts will still be satisfied.

In the case where this feature flag is activated, is it true that accounts created during this low rent regime could potentially be at risk of garbage collection if the storage cost parameter were to increase in the future?

Reply:
perhaps this idea can be developed in conjunction with account compression. Similar to your proposal: upon some grace period after a potential rent increase, an account with an insufficient deposit is compressed; instead of needing to top up their balance after 1 year they must decompress the account.